ember-phone-input/rollup.config.mjs

```diff
@@ -15,12 +18,20 @@ export default {
   plugins: [
     // These are the modules that users should be able to import from your
     // addon. Anything not listed here may get optimized away.
-    addon.publicEntrypoints(["index.js", "**/*.js"]),
+    addon.publicEntrypoints([
+      'components/**/*.js',
+      'services/**/*.js',
+      'instance-initializers/**/*.js'
```

Comment:
What is the `instance-initializers` folder about? Just asking because I'm wondering if need to add it here

Reply:
It loads up the phone input service, gets exported and imported in test-app, so that's why I thought we need it here

Reply:
As discussed during our sync, let's keep it in the public entry points for now 👌